cronex.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_runtime_dependency 'tzinfo'
-  spec.add_runtime_dependency 'unicode'
+  spec.add_runtime_dependency 'unicode', '~> 0.4.4.5'
```

Comment:
I'd go with `>= 0.4.4.5`

Reply:
Updated @alpinweis thanks for the feedback.